src/lib/core/components/computations/ComputationInstance.tsx

```diff
@@ -102,14 +108,20 @@ interface AppTitleProps {
   computation: Computation;
   computationAppOverview: ComputationAppOverview;
   condensed: boolean;
+  description:
```

Comment:
Since we expect the description to handle more than data and a single method, maybe `getAppDescription` or `useAppPropertiesForDisplay` could return a description string instead of a method and displayName?